onnx/defs/math/defs.cc

```diff
@@ -2020,43 +1922,35 @@ ONNX_OPERATOR_SET_SCHEMA(
             true,
             1,
             OpSchema::NonDifferentiable)
+        .TypeConstraint("TS", {"tensor(float)", "tensor(float16)", "tensor(bfloat16)"}, "Constrain scales.")
         .TypeConstraint(
             "T1",
-            {"tensor(int8)", "tensor(uint8)"},
+            {"tensor(int8)",
+             "tensor(uint8)",
+             "tensor(float8e4m3fn)",
+             "tensor(float8e4m3fnuz)",
+             "tensor(float8e5m2)",
+             "tensor(float8e5m2fnuz)"},
```

Comment:
Suggest changing the line below to "The type of input a and its zeropoint.". It is no longer just 8-bit integer. But no point in repeating the list of all types, I think it is simpler to use the description to say what inputs type is being described. Same for input b and output.

Reply:
done